frontend/tailwind.css

```diff
@@ -2,6 +2,54 @@
 @tailwind base;
 /* purgecss end ignore */
 
+html {
+    @apply text-base text-font-primary;
+}
+
+@screen lg {
+    html {
+        @apply text-lg-base;
+    }
+}
+
+body {
+    @apply text-body;
+}
+
+a {
+    @apply text-link-primary;
+}
+
+h1 {
+    @apply text-large-1;
+}
+
+h2 {
+    @apply text-large-2;
+}
+
+h3 {
+    @apply text-large-3;
+}
+
+h4 {
+    @apply text-large-4;
+}
+
+.post-content > .wp-block-group,
+.post-content > .wp-block-columns
+{
+    @apply py-block-y;
+}
+
+.container {
+    @apply max-w-container px-container-padding mx-auto; 
+}
+
+.container-narrow {
+    @apply max-w-container-narrow px-container-padding mx-auto; 
```

Comment:
the narrow container width from the post single page is usually x number of columns from the main container

Reply:
Yes, it is 8/12 columns. How would you represent this?
I'm actually wondering if we need to include the bootstrap column plugin by default, this might make things easier.